frontend/src/v5/ui/controls/assigneesSelect/assigneesSelect.component.tsx

```diff
@@ -89,26 +91,28 @@ export const AssigneesSelect = ({
 		</AssigneesListContainer>
 	);
 	return (
-		<SearchContextComponent fieldsToFilter={['_id', 'firstName', 'lastName', 'job', 'notFoundName']} items={allJobsAndUsersToDisplay}>
-			<AssigneesListContainer onClick={handleOpen} className={className}>
-				<AssigneesSelectMenu
-					open={open}
-					value={value}
-					onClose={handleClose}
-					onOpen={handleOpen}
-					disabled={disabled}
-					multiple={multiple}
-					isInvalid={(v) => invalidValues.includes(v)}
-					onChange={handleChange}
-					{...props}
-				/>
-				<AssigneesValuesDisplay
-					value={valueAsArray}
-					maxItems={maxItems}
-					showEmptyText={showEmptyText}
-					disabled={disabled || !showAddButton}
-				/>
-			</AssigneesListContainer>
-		</SearchContextComponent>
+		<FormControl required={false} disabled={disabled} error={props.error} className={className}>
+			<SearchContextComponent fieldsToFilter={['_id', 'firstName', 'lastName', 'job', 'notFoundName']} items={allJobsAndUsersToDisplay}>
+				<AssigneesListContainer onClick={handleOpen}>
+					<AssigneesSelectMenu
+						open={open}
+						value={value}
+						onClose={handleClose}
+						onOpen={handleOpen}
+						multiple={multiple}
+						isInvalid={(v) => invalidValues.includes(v)}
+						onChange={handleChange}
+						{...props}
+					/>
+					<AssigneesValuesDisplay
+						value={valueAsArray}
+						maxItems={maxItems}
+						showEmptyText={showEmptyText}
+						disabled={disabled || !showAddButton}
+					/>
+				</AssigneesListContainer>
+				<FormHelperText>{helperText}</FormHelperText>
```

Comment:
I don't get what this (and the whole FormControl) are for. I tried to tweak the props to add an error as well as an helper text and this is what it looks like

![image](https://github.com/user-attachments/assets/bdb0710a-4984-4137-b371-198d5439df4e)

Also, `AssigneesSelectMenu` is already wrapped in a `FormControl` at `Select` level, so I think if you need to add a new FormControl, It should only wrap `AssigneesValuesDisplay`